### ISSUE TITLE
Load more channels

### ISSUE THREAD

**Issue by mishkeTz** [GetStream/stream-chat-react-native#1032]
GetStream has limit of showing 30 channels in the list. I tried to implement load more button following this article https://getstream.io/chat/docs/sdk/reactnative/guides/customize-channel-list/#replacing-infinite-scroll-pagination-with-load-more-button.

I have set the following options on ChannelList:
`options={{ watch: true }}`

If I pass custom load more component then I still have old behaviour and the button is not rendered. However if I update the options prop and pass `limit: 10` then it will show the load button, when I click it once it will load more channels but then `hasNextPage` will be false in useChannelsContext - but in fact I know that the user is a member of 50+ channels.

My goal is to load more channels as long as user has any/is a member of channels that are not yet rendered, ideally that should work with `onEndReached` prop and not as a custom prop with 'load more' button.
What is the correct approach here, and was there a special reason why GetStream limited channels list to 30?

gz#17109

**Comment by vishalnarkhede** [comment]
Hey @mishkeTz its a bit hard to follow the issue from the written description. It would be really helpful to record a short loom video or something to demonstrate the issue please.
